.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
```

Comment:
does `nfm init` generate this as well or a limited version? if not i think that would be a good enhancement.

Reply:
no, i added circleci